Sming/system/m_printf.cpp

```diff
@@ -7,7 +7,8 @@ Descr: embedded very simple version of printf with float support
 */
 
 #include <stdarg.h>
-#include "../../SmingCore/SmingCore.h"
+//#include "../../SmingCore/SmingCore.h"
```

Comment:
@ADiea If you want to use only osapi.h then please remove the commented include above.